Fix unit test failures in Travis CI builds

Fix unit test failures in Travis CI builds, i.e.
```
[ERROR] Tests run: 13, Failures: 2, Errors: 1, Skipped: 0, Time elapsed: 0.014 s
<<< FAILURE! - in org.biojavax.SimpleNamespaceTest
[ERROR] testGetAuthority(org.biojavax.SimpleNamespaceTest)  Time elapsed: 0.007 s 
<<< FAILURE!
junit.framework.ComparisonFailure: expected:<respect mah authoritaaah!!!> but was:<null>
	at org.biojavax.SimpleNamespaceTest.testGetAuthority(SimpleNamespaceTest.java:130)
[ERROR] testGetAcronym(org.biojavax.SimpleNamespaceTest)  Time elapsed: 0 s
<<< FAILURE!
junit.framework.ComparisonFailure: expected:<by any other name> but was:<null>
	at org.biojavax.SimpleNamespaceTest.testGetAcronym(SimpleNamespaceTest.java:121)
[ERROR] testGetURI(org.biojavax.SimpleNamespaceTest)  Time elapsed: 0 s
<<< ERROR!
java.lang.NullPointerException
	at org.biojavax.SimpleNamespaceTest.testGetURI(SimpleNamespaceTest.java:164)
```

--- a/core/src/test/java/org/biojavax/SimpleNamespaceTest.java
+++ b/core/src/test/java/org/biojavax/SimpleNamespaceTest.java
@@ -116,9 +116,13 @@ public class SimpleNamespaceTest extends TestCase {
      */
     public void testGetAcronym() {
         System.out.println("testGetAcronym");
-        //System.out.println(ns.getAcronym());
-        //should be set because of retreival from the LRU cache
-        assertEquals(acronym, ns.getAcronym());
+
+        try{
+            ns.setAcronym(acronym);
+            assertEquals(acronym, ns.getAcronym());
+        }catch(ChangeVetoException ex){
+            fail("Was not expecting "+ex.getClass().getName());
+        }
     }
 
     /**
@@ -126,8 +130,13 @@ public class SimpleNamespaceTest extends TestCase {
      */
     public void testGetAuthority() {
         System.out.println("testGetAuthority");
-        
-        assertEquals(authority,ns.getAuthority());
+
+        try{
+            ns.setAuthority(authority);
+            assertEquals(authority, ns.getAuthority());
+        }catch(ChangeVetoException ex){
+            fail("Was not expecting "+ex.getClass().getName());
+        }
     }
 
     /**
@@ -158,10 +167,16 @@ public class SimpleNamespaceTest extends TestCase {
     /**
      * Test of getURI method, of class org.biojavax.SimpleNamespace.
      */
-    public void testGetURI() {
+    public void testGetURI() throws Exception {
         System.out.println("testGetURI");
-        
-       assertEquals(uriString, ns.getURI().toString());
+
+        URI uri = new URI(uriString);
+        try{
+            ns.setURI(uri);
+            assertEquals(uriString, ns.getURI().toString());
+        }catch(ChangeVetoException ex){
+            fail("Was not expecting "+ex.getClass().getName());
+        }
     }
 
     /**

